docs/start/query.md

```diff
@@ -0,0 +1,14 @@
+# Querying data
+
+SQL is the most widely used language for querying data and is the natural
+choice for people in many roles working with data in databases.
+
+CrateDB extends industry-standard SQL with functionalities to support its
+data types, data I/O procedures, and cluster management.
+
+To learn more, check out the {ref}`SQL syntax reference manual <crate-reference:sql>`.
+
+
+:::{note}
+🚧 _Please note this page is a work in progress._ 🚧
```

Comment:
I would prefer we don't merge these kind of "WIP" before they are done - i.e. no longer are WIP.